homage-core/src/main/java/me/oriley/homage/Homage.java

```diff
@@ -192,17 +196,21 @@ public void refreshLibraries() {
             library.setLicense(license);
 
             int iconRes = -1;
-            String icon = library.getLibraryIcon();
-            if (!TextUtils.isEmpty(icon)) {
-                iconRes = getResourceId(mContext, icon, ResourceType.DRAWABLE);
-                if (iconRes <= 0) {
-                    iconRes = getResourceId(mContext, icon, ResourceType.MIPMAP);
-                }
-                if (iconRes <= 0) {
-                    iconRes = android.R.drawable.sym_def_app_icon;
+            String iconUri = library.getLibraryIcon();
+            if (!TextUtils.isEmpty(iconUri)) {
+                if (Patterns.WEB_URL.matcher(iconUri).matches()) {
```

Comment:
This will only match web related Uri's. Could probably be as simple as doing `if (icon.contains("://"))` to match for any Uri, and leave it to the user/adapter side to handle whichever Uri's they may have decided to use.